telemetry/metrics/init.go

```diff
@@ -0,0 +1,117 @@
+package metrics
+
+import (
+	"context"
+	"math/rand"
+
+	"github.com/gnolang/gno/telemetry/exporter"
+	"github.com/gnolang/gno/telemetry/options"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc"
+	"go.opentelemetry.io/otel/metric"
+	sdkMetric "go.opentelemetry.io/otel/sdk/metric"
+)
+
+var (
+	ctx context.Context
+
+	// Metrics.
+	BroadcastTxTimer Int64Collector
+	BuildBlockTimer  Int64Collector
+)
```

Comment:
I'm not a huge fan of using global metrics objects as a funnel for the entire collection system -- I always prefer to directly manage these things and pass them down to modules that make sense to have access to them

Having said this, I realize we don't have any option right now to get this data, apart from manually parsing it ourselves based on node responses. I'm fine with keeping it as is (global) in this PR, but would love to discuss a more controlled system in the next iteration

Reply:
Okay, I'd be open to the conversation.  I've usually worked with the metrics defined in a package like this so the initialization is all handled at startup rather than being initialized every place they are used.  This PR is mainly concerned with enabling metrics collection.  I'm not even sure the collection points are 100% correct -- just examples for now.